### PR TITLE
Task4

### PR DESCRIPTION
**Pull request by mnasevich**
Здравствуйте, Степан. Отправляю 4 задание.

### CODE REVIEW

File: task4/Dog_API/Описание_Запроса_1.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+������� ������:
```
Comment (smiakchilo) [inline task4/Dog_API/Описание_Запроса_1.txt:1]: Please **always** store files with names in Latin. 
Please **always** write whatever technical documentation, comments, desciptions in English. There is no other way to do this in a commercial project.

Storing comments in Russian makes them unreadable like this. This is actually a *huge* minus)
image

**Review by smiakchilo — COMMENTED** [review]
Concerning request descriptions, please try to master use the "technical" style of comments. Learn how `Javadoc` comments generally look like (in frontend, there's the equivalent `jsdoc` standard.
What was expected, is something like
 ```
api_variable -- Used to specify the id of image to display
api_variable_2 -- Used to set the format of images (either png or jpg)
```

**Also**, please never mix up different tasks into one pull request. I see markup design here which was probably developed for the *Task 3*.
Starting a new task, you must leave alone the current branch, and switch to the "basic" branch (like *master*) and create a new branch from there

**Comment by mnasevich** [comment]
Спасибо.Я учту замечание и в дальнейшем буду стараться делать так, как следует. Скажите, данное задание засчитано?

**Comment by smiakchilo** [comment]
Да)

**Pull request merged**